i18n/zh/docusaurus-plugin-content-docs/current/user-manuals/iot/edgex-foundry.md

```diff
@@ -3,7 +3,7 @@ title: 云原生管理端设备
 ---
 
 
-本文档主要讲述如何在已有的OpenYurt集群上安装Yurt-Device-Controller 和 Yurt-EdgeX-Manager组件，并通过部署虚拟端设备来展示如何通过云原生的方式管理边缘端设备。
+本文档主要讲述如何在已有的OpenYurt集群上安装YurtIoTDock，并通过部署虚拟端设备来展示如何通过云原生的方式管理边缘端设备。
 
 对于有兴趣的读者，可以去相关的github仓库参考本文使用组件的具体实现：[Yurt-Device-Controller](https://github.com/openyurtio/yurt-device-controller) 
```

Comment:
Just to add to the current situation: `yurt-edgex-manager` and `yurt-device-controller` have been merged into the main repo, and then introduce the new crd and controller.